solidstart-tanstackquery-tailwind-modules/src/components/GistPanel/Gists.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { Router } from '@solidjs/router';
+import { render } from 'solid-testing-library';
+import { beforeEach, describe, expect, it } from 'vitest';
+import { GistsData } from './index';
+import { gists } from './data';
+
+describe('User gist card', () => {
+  let wrapper: any;
+
+  beforeEach(async () => {
+    wrapper = await render(() => (
+      <Router>
+        <GistsData gists={gists} />
+      </Router>
+    ));
+  });
```

Comment:
If we move the assignation of wrapper inside every single tests instead of using beforeEach (we only have a few tests, so it's not a big effort I think); we can use typescript's type inference in order to have wrapper type and not use `any`.

Reply:
I understand, but I think this is fine and it's a good practice and we don't need to do any render again.